action-server/actions/daily_ci_enroll_form.py

```diff
@@ -270,14 +275,41 @@ def submit(
         tracker: Tracker,
         domain: Dict[Text, Any],
     ) -> List[Dict]:
-        if tracker.get_slot(DO_ENROLL_SLOT) is True:
+        if tracker.get_slot(DO_ENROLL_SLOT) is False:
```

Comment:
Is this slot initialized to False? I just want to make sure that when we reach this point the slot can't be none.

Reply:
good point!